docs/accessing-elastic-services.asciidoc

```diff
@@ -229,7 +229,21 @@ Now you should get this message:
 curl: (51) SSL: no alternative certificate subject name matches target host name '35.198.131.115'
 ----
 
-Add the external IP of the service to the SANs of the certificate in the same Elasticsearch resource YAML manifest used to create the cluster and apply it again using `kubectl`.
+As the certificate subject by default is service name we can resolve this by using the service name and specify the resolve information through an additional argument.
```

Comment:
I think there might be a better way to phrase and avoid the double usage of "resolve". Maybe
>As the certificate subject by default is service name we can resolve this by using the service name and specify the resolve information through an additional argument.

to

>You can avoid this in curl by manually specifying the certificate subject and IP address to the `resolve` argument
?

Reply:
Heck, sorry I merged without seeing this comment.
We can bring this little improvement in another PR.